mautic/mautic
# Future browsers samesite none

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes, make Mautic ready for future browsers
| Automated tests included? |Yes, they work just with Xdebug enabled. Otherwise, they are skipped.
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: Modified CookieHelper in CoreBundle to send sameSite = None for secure connection.

#### Steps to test this PR:
1. Enable these flags in Chrome `chrome://flags/#same-site-by-default-cookies` , `chrome://flags/#cookies-without-same-site-must-be-secure`
2. Check that you get all cookies as before and tracking works well
